From 7899749a6ef41fe958de5bd618fc0b149305612a Mon Sep 17 00:00:00 2001
From: Richard Zhu <Richard.Zhu@freescale.com>
Date: Fri, 16 Jan 2015 15:00:54 +0800
Subject: [PATCH] MLK-10058-1 ARM: imx6sx: add one pcie phy regulator in gpc
 dts

Register one pcie phy regulator in gpc driver, so thus,
the gpc related operations when pcie phy is powered
on/off can be moved into gpc driver

Signed-off-by: Richard Zhu <Richard.Zhu@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index 79b9e5ec..fc3eb1b 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -808,6 +808,7 @@
 				interrupts = <GIC_SPI 89 IRQ_TYPE_LEVEL_HIGH>;
 				fsl,mf-mix-wakeup-irq = <0x7c00000 0x3d00 0x0 0x400240>;
 				clocks = <&clks IMX6SX_CLK_GPU>;
+				pcie-phy-supply = <&reg_pcie_phy>;
 				#power-domain-cells = <1>;
 			};
 
-- 
1.8.0

